src/libs/actions/Session/index.js

```diff
@@ -286,6 +287,12 @@ function signIn(password, validateCode, twoFactorAuthCode) {
         params.password = password;
     }
 
+    // If the user chooses a language other than the default, pass that to the API so it'll be set on their account upon sign in
+    // We only do this for non-default locales because that indicates they took an explicit action to set their language
```

Comment:
👍

Reply:
Yeah, the reason I did it this way was because I was imagining this scenario: 

- user signs into  account on web, sets account to spanish
- user downloads mobile app, lands on english sign in page
- doesn't change language on sign-in page because dropdown is hard to find and how to sign in is obvious

If we used the sign-in page language in that scenario, their account would be set back to english.

Reply:
Yeah it's a fair point. Ok I think what we'll do is let's just always pass the language from the sign in page in this PR. And then I'll open a separate issue for detecting the device language on the sign-in page. Appreciate the thoughts!